ccmlib/scylla_node.py

```diff
@@ -1127,13 +1127,11 @@ def update_yaml(self):
                 self.get_base_cassandra_version() >= 1.2):
             _, data['native_transport_port'] = self.network_interfaces['binary']
 
-        data['data_file_directories'] = [os.path.join(self.get_path(), 'data')]
+        data['workdir,W'] = self.get_path()
+        # The default path of commitlog subdirectory is workdir/commitlog.
+        # This option is set seperately from the workdir to keep backward compatibility with `commitlogs` value.
```

Comment:
s/seperately/separately

I'd also add why we need this backwards compatibility --- because some dtests depend on it